css/templates/travel/page-vars.css

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:root {
+  --font-color-primary: var(--black);
```

Comment:
you will need to set the secondary colors for this to work right

Reply:
👍 Cool, I'll set it

Reply:
also set color-primary and secondary

Reply:
https://github.com/ampproject/ampstart/blob/master/css/ampstart-base/base-page-vars.css - Look at this and set anything that you should (setting all is great :))

Reply:
Everything that we can use should be set now